public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_monitor/azure_monitor_datasource.ts

```diff
@@ -296,4 +299,29 @@ export default class AzureMonitorDatasource extends DataSourceWithBackend<AzureM
 
     return workingQuery;
   }
+
+  async getProvider(providerName: string) {
+    return await this.getResource<AzureMonitorProvidersResponse>(
+      `${routeNames.azureMonitor}/providers/${providerName}?api-version=2021-04-01`
```

Comment:
nit: for keeping the current style it may be good to put all api versions in the same place (line 41)

Reply:
At some point I'll do a broader review of API versions to see if we can find some common ones 😊